src/Model/Customer/CustomerBuilder.php

```diff
@@ -61,12 +61,12 @@ public function withPaymentMethod(PaymentMethod $paymentMethod): self
 
     public function build(): Customer
     {
-        $customer = (new Customer($this->get('id')))
-            ->setFirstName($this->get('firstName'))
-            ->setLastName($this->get('lastName'))
-            ->setEmailAddress($this->get('emailAddress'));
+        $customer = (new Customer($this->getId()))
+            ->setFirstName($this->data['firstName'] ?? '')
+            ->setLastName($this->data['lastName'] ?? '')
+            ->setEmailAddress($this->data['emailAddress'] ?? '');
 
-        $paymentMethods = $this->get('paymentMethods') ?? [];
+        $paymentMethods = $this->data['paymentMethods'] ?? [];
```

Comment:
maybe `getPaymentMethods()` is a function that returns an array? maybe that's just overkill

Reply:
Yeah. I'm leaning towards overkill 🙇